src/renderer/components/Modal/ModalPeer.vue

```diff
@@ -146,10 +146,12 @@ export default {
   },
 
   methods: {
-    emitConnect () {
-      this.$emit('connect', {
-        peer: this.form,
-        closeTrigger: this.closeTrigger
+    async validate () {
```

Comment:
The `validate` method is not linked to the connect button. I received the message on the browser console:

```
Property or method "emitConnect" is not defined on the instance but referenced during render...
```

Although I have updated the code, the modal does not close and does not show any alerts if the connection is successful or not.